index.js

```diff
@@ -642,7 +665,7 @@ Compiler.prototype = {
     if (code.buffer) {
       var val = code.val.trim();
       val = 'null == (jade_interp = '+val+') ? "" : jade_interp';
-      if (code.mustEscape !== false) val = this.runtime('escape') + '(' + val + ')';
+      if (code.mustEscape !== undefined ? code.mustEscape : code.escape) val = this.runtime('escape') + '(' + val + ')';
```

Comment:
I'd prefer to stick to always escaping unless `mustEscape === false` rather than aiming for compatibility with out of date parsers.